chainnotif/bitcoindnotify/bitcoind.go

```diff
@@ -818,7 +843,8 @@ func (b *BitcoindNotifier) RegisterSpendNtfn(outpoint *wire.OutPoint,
 		}
 
 		if uint32(blockHeight) > ntfn.HistoricalDispatch.StartHeight {
-			ntfn.HistoricalDispatch.StartHeight = uint32(blockHeight)
+			ntfn.HistoricalDispatch.StartHeight =
+				uint32(blockHeight)
```

Comment:
nit: IMO we can treat casts (and the use of `make()`) like normal function calls and format accordingly.
So this could be:
```
			ntfn.HistoricalDispatch.StartHeight = uint32(
				blockHeight,
			)
```
But non-blocking, more of an FYI.